src/inputs/Button/Button.tsx

```diff
@@ -1,8 +1,41 @@
-import { PropsWithChildren } from "preact/compat";
-import { h } from "preact";
+import React from 'react'
+import { PropsWithChildren } from 'react'
+import { h } from 'preact'
+import styles from './Button.module.css'
+import ButtonStories, { Primary } from './Button.stories'
+// import { ikona } from '../../assets/color.svg'
 
-interface ButtonProps {}
+interface ButtonProps {
+	type?: 'button' | 'submit'
+	disabled?: boolean
+	icon?: React.ReactElement
+	isLoading?: boolean
+	onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+	variant: 'primary' | 'secondary'
+}
 
-export const Button = (props: PropsWithChildren<ButtonProps>) => {
-  return <button>{props.children}</button>;
-};
+const Button: React.FC<PropsWithChildren<ButtonProps>> = ({
```

Comment:
**suggestion:** If you use `React.FC`, we don't have to use `PropsWithChildren` anymore. So it can be:  

``const Button: React.FC<ButtonProps> ``